src/activities/hoc2019/PondCreator.jsx

```diff
@@ -8,6 +8,11 @@ const P5 = require('../../utils/loadP5');
 
 const FISH_COUNT = 9;
 
+export const Classifiers = Object.freeze({
```

Comment:
really nit: isn't `Classes` a bit word for this than `Classifiers`? I _think_ classifiers are the things that actually do the classifying?

Reply:
lol i'm not sure anymore but i like `Classes`!

Reply:
Lol yeah I'm definitely not sure about any of the terminology...

Reply:
maybe i'll do `ClassType` for now so that it isn't ambiguous with the loaded keyword class